crates/ruma-events/src/secret_storage/key.rs

```diff
@@ -54,6 +54,9 @@ fn is_default_bits(val: &UInt) -> bool {
 }
 
 /// A key description encrypted using a specified algorithm.
+///
+/// The only algorithm currently specified is `m.secret_storage.v1.aes-hmac-sha2`, so this
+/// essentially represents `AesHmacSha2KeyDescription` in the [spec](https://spec.matrix.org/v1.9/client-server-api/#msecret_storagev1aes-hmac-sha2).
```

Comment:
I thought this extra clarification might be helpful, since it wasn't clear to me exactly what this struct represented previously. Not sure if it's in keeping with ruma's style though?

Reply:
Yeah that seems helpful, thanks.

I wonder why rustfmt isn't complaining about the rather long line though. Can you put the spec link on its own line (either including the link text, or as a `[spec]: https://...` link definition outside the paragraph)?

Reply:
```suggestion
/// The only algorithm currently specified is `m.secret_storage.v1.aes-hmac-sha2`, so this
/// essentially represents `AesHmacSha2KeyDescription` in the 
/// [spec](https://spec.matrix.org/v1.9/client-server-api/#msecret_storagev1aes-hmac-sha2).
```